scripts/lib/zulip_tools.py

```diff
@@ -355,35 +355,19 @@ def parse_lsb_release():
             vendor = 'RedHat'
             codename = 'rhel' + info[6][0]  # 7
         distro_info = dict(
-            DISTRIB_CODENAME=codename,
-            DISTRIB_ID=vendor,
+            VERSION_ID=codename,
+            ID=vendor,
             DISTRIB_FAMILY='redhat',
         )
         return distro_info
-    try:
-        # For performance reasons, we read /etc/lsb-release directly,
-        # rather than using the lsb_release command; this saves ~50ms
-        # in several places in provisioning and the installer
-        with open('/etc/lsb-release', 'r') as fp:
-            data = [line.strip().split('=') for line in fp]
-        for k, v in data:
-            if k not in ['DISTRIB_CODENAME', 'DISTRIB_ID']:
-                # We only return to the caller the values that we get
-                # from lsb_release in the exception code path.
-                continue
-            distro_info[k] = v
-        distro_info['DISTRIB_FAMILY'] = 'debian'
-    except FileNotFoundError:
-        # Unfortunately, Debian stretch doesn't yet have an
-        # /etc/lsb-release, so we instead fetch the pieces of data
-        # that we use from the `lsb_release` command directly.
-        vendor = subprocess_text_output(["lsb_release", "-is"])
-        codename = subprocess_text_output(["lsb_release", "-cs"])
-        distro_info = dict(
-            DISTRIB_CODENAME=codename,
-            DISTRIB_ID=vendor,
-            DISTRIB_FAMILY='debian',
-        )
+    with open('/etc/os-release', 'r') as fp:
+        for line in fp:
```

Comment:
Although I haven’t actually seen any, the spec says we may have to deal with blank lines and comments: `line = line.strip(); if not line or line.startswith('#'): continue`.

Reply:
Added in 7th commit.

Reply:
7th commit updated and force-pushed.